tests/filecheck/dialects/func/func_invalid.mlir

```diff
@@ -31,3 +31,9 @@ builtin.module {
 }
 
 // CHECK: Expected all arguments to be named or all arguments to be unnamed.
+
+// -----
+
+"func.call"() { "callee" = @call::@invalid } : () -> ()
```

Comment:
This and other parts of this PR seem to have crept up from https://github.com/xdslproject/xdsl/pull/1598.
Could you please separate the two?

Reply:
Yes so sorry I messed up with some branch, gonna solve that now!